ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldesfragment/services/RefragmentationServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.entities.LdesFragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.entities.Member;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.member.repository.MemberRepository;
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import org.springframework.stereotype.Service;
+
+import java.util.stream.Stream;
+
+@Service
+public class RefragmentationServiceImpl implements RefragmentationService {
+	private final MemberRepository memberRepository;
+	private final ObservationRegistry observationRegistry;
+
+	public RefragmentationServiceImpl(MemberRepository memberRepository, ObservationRegistry observationRegistry) {
```

Comment:
Maybe we can use the new architecture instead of calling the deprecated memberRepository? We could ask the ingester for his members?

Reply:
Was doubting here, but then I thought it would be easier to now test it with the old MemberRepository and replace this code in: https://vlaamseoverheid.atlassian.net/browse/VSDSPUB-655